python/cuml/cluster/kmeans.pyx

```diff
@@ -60,6 +60,7 @@ cdef extern from "cuml/cluster/kmeans.hpp" namespace "ML::kmeans":
                           const float *X,
                           int n_samples,
                           int n_features,
+                          const float *sample_weight,
```

Comment:
Does cuML project not guarantee stability in its C API? I've seen issues with changing/breaking C API in other projects like XGBoost.

Reply:
Nope, no stability guarantees at this time. After we get to 1.0 we will stabilize the API. But a good question!